src/Sql/Sql/Instance Pools/Cmdlet/GetAzureSqlInstancePool.cs

```diff
@@ -0,0 +1,127 @@
+﻿using System.Collections.Generic;
```

Comment:
@johnpaulkee it looks like a lot of the `.cs` files you've added are missing license headers -- please make sure to add this to the top of each of these files

Reply:
Will add license headers for these files.